ios/browser/api/brave_shields/adblock_filter_list_catalog_entry.mm

```diff
@@ -15,9 +15,13 @@ @interface AdblockFilterListCatalogEntry ()
 @property(nonatomic, copy) NSString* title;
 @property(nonatomic, copy) NSArray<NSString*>* languages;
 @property(nonatomic, copy) NSString* supportURL;
+@property(nonatomic, copy) NSString* desc;
```

Comment:
we're duplicating this stuff here? That doesn't seem right, we should revisit this in a follow-up

Reply:
Not sure if you're commenting on the wrapper itself or property being "duplicated", no way around the former to make it exposed to Swift, for the latter the property in the header is defined as `readonly`, redefined in the source for mutability (standard Obj-C practice)